test/combinations.jl

```diff
@@ -178,6 +184,9 @@ end
     end
     solution = 2π * u"Ω*m"
 
+    # Enzyme support
+    @test MeshIntegrals.supports_autoenzyme(curve) == false
```

Comment:
Would these tests maybe be better located in the utils.jl tests?

Reply:
In my head it makes more sense to group all things relevant to a specific geometry with each other, but if you think we should move it to utils.jl, I will. I don't feel strongly about it!

Reply:
Yeah, I don't know. I also do not care so much. We can also keep it here if @mikeingold does not object.

Reply:
If we're only implementing a smallish number of these tests, then I actually would lean towards moving these tests into `utils.jl` under the `@testitem "Differentiation"`, maybe in their own `@testset` if you think it makes sense. If this property needed to be checked for every geometry type, then maybe we could slide it into the test generation function based on `supports.autoenzyme`.

Prior to the recent refactor of `combinations.jl`, I'd had the same idea to add a few extra tests into these sections wherever they were most convenient, but I found that it made keeping track of what tests were implemented and where more convoluted. In the long-run I think the more organized and structured we can make the tests the better.